src/public/index.html

```diff
@@ -60,6 +60,18 @@ <h1 id="rpsStat">-</h1>
       <canvas id="rpsChart" width="200" height="100"></canvas>
     </div>
   </div>
+  <div class="container">
+    <div class="stats-column">
+      <h5>Status Codes</h5>
+      <h6 style="color: #75D701">2xx</h6>
+      <h6 style="color: #47b8e0">3xx</h6>
+      <h6 style="color: #ffc952">4xx</h6>
+      <h6 style="color: #E53A40">5xx</h6>
```

Comment:
Hi @RafalWilinski I will have a look at this ASAP and give you my feedback.
Thanks,
On Mon, 12 Sep 2016 at 12:40, Rafal Wilinski <notifications@github.com>
wrote:

> In src/public/index.html
> <https://github.com/RafalWilinski/express-status-monitor/pull/52#discussion_r78440240>
> :
>
> > @@ -60,6 +60,18 @@ <h1 id="rpsStat">-</h1>
> >        <canvas id="rpsChart" width="200" height="100"></canvas>
> >      </div>
> >    </div>
> > +  <div class="container">
> > +    <div class="stats-column">
> > +      <h5>Status Codes</h5>
> > +      <h6 style="color: #75D701">2xx</h6>
> > +      <h6 style="color: #47b8e0">3xx</h6>
> > +      <h6 style="color: #ffc952">4xx</h6>
> > +      <h6 style="color: #E53A40">5xx</h6>
>
> Inline styles are bad, I know, I'm lazy. Also, i think there's better
> solution to unify them with these in JS file
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/RafalWilinski/express-status-monitor/pull/52/files/8e2b695cd5c800fe3d0de18a40a9c0c11ea69318#r78440240>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGIC3dAIgOyllYNvr96f-tFtYeMUUcfiks5qparKgaJpZM4J6wVB>
> .
>
-- 
In theory there is no difference between theory and practice. In practice
there is.

*Jan L. A. van de Snepscheut*